projects/console-web-front/src/components/hosts/HostListController.tsx

```diff
@@ -153,6 +166,21 @@ const HostItem = ({ host }: HostItemProps) => {
       ),
       key: 'token',
     },
+    {
+      label: (
+        <MenuItemButton
+          danger
+          onConfirm={handleHostAppUpdate}
+          modalTitle={t('device-farm:hostItemUpdateMenu')}
+          modalButtonTitle={t('device-farm:hostUpdateModalButtonText')}
+          modalContent={<StyledDeleteModalContent>{t('device-farm:hostUpdateModalContentInfo')}</StyledDeleteModalContent>}
+          confirmButtonId="host-update-confirm-btn"
+        >
+          {t('device-farm:hostItemUpdateMenu')}
+        </MenuItemButton>
+      ),
+      key: 'update',
+    },
```

Comment:
Displaying this menu when the host is updatable could be more friendly..!

Reply:
There is api to get latest apps {{API_BASE_URL}}/downloads/dogu-agent/latest! https://github.com/dogu-team/dogu/pull/319/files#diff-3d51b5a5e5b7b359f659f7bd551616df75e3e82d289cecb10f0293a40e9f4b08